Index: src/main/java/org/example/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.example;\r\n\r\nimport org.springframework.context.ApplicationContext;\r\nimport org.springframework.context.annotation.AnnotationConfigApplicationContext;\r\n\r\n// Spring core Annotations\r\n//component\r\n//Autowired\r\n//Configuration\r\n//primary used to mention if collisions occurs which need to execute first\r\n//Component Scan is used to scan entire mentioned directory or path\r\n//Quantifer used to mention the name which is going to execute if called\r\n\r\npublic class Main {\r\n\r\n\r\n    public static void main(String[] args) {\r\n        ApplicationContext factory=new AnnotationConfigApplicationContext(AppConfig.class);\r\n\r\n        Teacher obj=factory.getBean(Teacher.class);\r\n        obj.Display();\r\n\r\n    }\r\n }
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/example/Main.java b/src/main/java/org/example/Main.java
--- a/src/main/java/org/example/Main.java	(revision c1a19afd1243bcefb36966dcf51460a14637b483)
+++ b/src/main/java/org/example/Main.java	(date 1674760199274)
@@ -19,6 +19,7 @@
 
         Teacher obj=factory.getBean(Teacher.class);
         obj.Display();
+        //THis is spring application Demo
 
     }
  }
\ No newline at end of file
